nbs/14_display.ipynb

```diff
@@ -63,7 +63,7 @@
     "    Calculate the percentage of a target protein covered by a list of peptides.\n",
     "    Args:\n",
     "        target_sequence (str): the protein sequence against which the peptide_list should be compared.\n",
-    "        peptide_list (str): the list of peptides (str) to be compared against the target_sequence.\n",
+    "        peptide_list (list(str)): the list of peptides (str) to be compared against the target_sequence.\n",
```

Comment:
Phython type hints are still a bit magic for me and this is probably very nitty gritty but I would think that a list of string should to be written as:
```
from typing import List
peptide_list (List[str])
```

Reply:
Thanks, yes the correct syntax was not exactly clear to me :)

Reply:
Thanks - the correct syntax there was not exactly clear to me